### PR TITLE
Refactor Logotype to be Reusable in Footer

### PR DESCRIPTION
**Pull request by martypowell**
Purpose of this pr is to make the "Brand Text" resuable in the header and footer. The header text is link, while the footer text is not.

- Removed link from `LogoType` and moved it to the `SiteHeader`
- Added title and fixed href in `SiteHeader`
- Renamed `LogoType` to `BrandText` to make sure more sense to both header and footer.
- Exported missing component `Fieldset`

### CODE REVIEW

File: src/styles/partials/components/_brand-text.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.dg_logotype {
+.dg_brand-text {
   border-right: $border-width-small solid $gray-light;
   padding: 0 $padding-large 0 $padding-largest;
   text-decoration: none;
```
Comment (martypowell) [inline src/styles/partials/components/_brand-text.scss]: this can be removed (i think).

Comment (martypowell) [reply src/styles/partials/components/_brand-text.scss]: removed.

**Review by tmccoy529 — DISMISSED** [review]
This looks good to me

**Comment by martypowell** [comment]
No version, bump, will do now.

**Pull request merged**